ledfx/playlists.py

```
@@ -0,0 +1,648 @@
+                    playlist.get("default_duration_ms", 500),
+                )
+
+                # Determine jitter and effective duration
+                timing = playlist.get("timing", {}) or {}
+                jitter = timing.get("jitter", {}) or {}
+                jitter_enabled = bool(jitter.get("enabled", False))
+                if jitter_enabled:
+                    fmin = float(jitter.get("factor_min", 1.0))
+                    fmax = float(jitter.get("factor_max", 1.0))
+                    # sample factor uniformly
+                    factor = random.uniform(fmin, fmax)
```

Ruff (0.13.1)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)